js/color-maps.ts

```diff
@@ -0,0 +1,32 @@
+import config, { Colormap } from "./config";
+import PlanetCrustAgePNG from "../images/planet-crust-age/planet-crust-age@3x.png";
+import PlanetPlateColorPNG from "../images/planet-plate-color/planet-plate-color@3x.png";
+import PlanetRockTypesPNG from "../images/planet-rock-types/planet-rock-types@3x.png";
+import PlanetTopographicPNG from "../images/planet-topographic/planet-topographic@3x.png";
+
+const COLORMAP_OPTIONS: { label: string, value: Colormap, image: any }[] = [
+  { value: "topo", label: "Topographic", image: PlanetTopographicPNG },
+  { value: "plate", label: "Plate Color", image: PlanetPlateColorPNG },
+  { value: "age", label: "Crust Age", image: PlanetCrustAgePNG },
+  { value: "rock", label: "Rock Type", image: PlanetRockTypesPNG }
+];
+
+export function getColorMapImage(colorMap: Colormap) {
+  return COLORMAP_OPTIONS.find(item => item.value === colorMap)?.image;
+}
+
+export function getColorMapLabel(colorMap: Colormap) {
+  return COLORMAP_OPTIONS.find(item => item.value === colorMap)?.label;
+}
+
+export function getAvailableColorMaps(_config: Record<string, any>) {
```

Comment:
Because there's a module-level imported `config` and the linter doesn't allow shadowed names. We could use another name, of course (`localConfig`, `paramConfig`, etc.) but this seems like a simple convention for disambiguation.

Reply:
I see. When I asked before, it wasn't clear to me why we need `_config` or `localConfig`, or `paramConfig` at all. But then I realized these functions are called from the authoring page with the modified config (its state).